doc/tutorial/single_cell_bluepyopt.rst

```diff
@@ -0,0 +1,288 @@
+      "gImbar_Im.apical": 0.000143,
+      "gNaTa_tbar_NaTa_t.axonal": 3.137968,
+      "gK_Tstbar_K_Tst.axonal": 0.089259,
+      "gamma_CaDynamics_E2.axonal": 0.002910,
+      "gNap_Et2bar_Nap_Et2.axonal": 0.006827,
+      "gSK_E2bar_SK_E2.axonal": 0.007104,
+      "gCa_HVAbar_Ca_HVA.axonal": 0.000990,
+      "gK_Pstbar_K_Pst.axonal": 0.973538,
+      "gSKv3_1bar_SKv3_1.axonal": 1.021945,
+      "decay_CaDynamics_E2.axonal": 287.198731,
+      "gCa_LVAstbar_Ca_LVAst.axonal": 0.008752,
+      "gamma_CaDynamics_E2.somatic": 0.000609,
+      "gSKv3_1bar_SKv3_1.somatic": 0.303472,
+      "gSK_E2bar_SK_E2.somatic": 0.008407,
+      "gCa_HVAbar_Ca_HVA.somatic": 0.000994,
+      "gNaTs2_tbar_NaTs2_t.somatic": 0.983955,
+      "decay_CaDynamics_E2.somatic": 210.485284,
+      "gCa_LVAstbar_Ca_LVAst.somatic": 0.000333
+   }
+
+We can now export the model to a given target directory using the ``create_acc`` module in BluePyOpt.
+
+.. code-block:: python
+
+   ephys.create_acc.output_acc('path/to/exported/l5pc', cell_model, param_values)
+
+This exports a model in a mixed JSON/ACC format analogous to that in the example directory of Arbor at ``python/example/single_cell_bluepyopt/l5pc``. The whole set of steps to construct and export the model is automated in the script `generate_acc.py <https://github.com/BlueBrain/BluePyOpt/blob/master/examples/l5pc/generate_acc.py>`_.
+
+This script also supports axon replacement by first instantiating the morphology in BluePyOpt through the Neuron simulator before exporting the model to JSON/ACC with
+
+.. code-block:: python
+
+   nrn_sim = ephys.simulators.NrnSimulator()
+   cell_model.instantiate_morphology_3d(nrn_sim)
+
+The model in the example directory ``python/example/single_cell_bluepyopt/l5pc`` was exported using this setting. For this reason, we find an axon-replacement morphology and a modified morphology (after performing axon replacement on the original) in the ACC format in that directory. The latter can be obtained by setting ``create_mod_acc=True`` as a parameter to ``create_acc``. For a more basic example, consider ``python/example/single_cell_bluepyopt/simplecell``, where no axon replacement is performed and, hence, only the original morphology is exported.
+
+Load from JSON/ACC
+------------------
+
+The exported model is summarized by a JSON file in the output directory, that references a morphology file (for the layer-5 pyramical cell in ASC, for the simple-cell in SWC), a label dictionary and decor (both in ACC) with relative paths. For the simple-cell example, this amounts to
+
+.. literalinclude:: ../../python/example/single_cell_bluepyopt/simplecell/simple_cell.json
+   :language: json
+   :dedent:
+
+In case of axon replacement, also the axon-replacement morphology and optionally the morphology after replacing the axon in the original are exported, as in the layer-5 pyramidal cell example.
+
+.. literalinclude:: ../../python/example/single_cell_bluepyopt/l5pc/l5pc.json
```

Comment:
I've tried to add more references to the included JSON format, which should make it clear. Indeed, when used in natural language the terms can be confusing. Let me know if the current situation is satisfactory. Otherwise, we can improve it again.

Reply:
That's better, thanks! On tiny remaining comment: move the 'also' after the comma further up ('are _also_ exported in the ACC format'). It's not incorrect, but it is a very long clause that you've stuck in the middle ;)

Reply:
Done in 613c1c0.